### PR TITLE
Export 2018 03 13 (Rocket 1.0.5) 

### PR DESCRIPTION
**Pull request by cnevinc**
Release: 1.0.5
Targeted deadline: Mar 14th

### CODE REVIEW

**Comment by Delphine** [comment]
Hi - I'm sorry but a timeline of one day for localization is not realistic. Strings need to be translated and then we need to have more time for testing them on screenshots - and then fixing anything if needed.
I'll review this and merge, but please understand these might not get in in time, and that we can't usually expect a 2day window for localization work. Thanks

File: locales/templates/app.pot [hunk 1]
```diff
@@ -763,10 +763,18 @@ msgctxt "rate_app_dialog_btn_go_rate"
 msgid "Yes, rate 5 stars!"
 msgstr ""
 
+msgctxt "rate_app_notification_action_rate"
+msgid "RATE 5 STAR"
```
Comment (Delphine) [inline locales/templates/app.pot]: Shouldn't this be "rate 5 starS"?

Comment (cnevinc) [reply locales/templates/app.pot]: Thanks! It should be 5 "stars".

Comment (cnevinc) [reply locales/templates/app.pot]: And also sorry for the deadline. It's my mistake.

Comment (Delphine) [reply locales/templates/app.pot]: @cnevinc: Ok can you commit again to fix this? Then i'll be able to merge. Thanks!

**Pull request merged**